documentation/getting_started/bug_bounty.md

```diff
@@ -1,74 +1,83 @@
 # UMA Bug Bounty Program
 
-Security of the platform is our highest priority. 
```

Comment:
@chrismaree I'm not sure why this `.md` file is appearing in my PR but I think its because Regina/Hart is not set up with the pre-commit linter so my PR, being the "most recent" is making some linting changes.

Reply:
@chrismaree should the linter be touching markdown files...?

Reply:
This is curious. I will take a look.

Reply:
So it looks like pritter is formatting everything automatically and This pr does not conform to the standard. If I include `*.md` in our `.prettierignore` it ignores it. This is a fine solution for now I think. I'm looking why the command in our `package.json` is linting this anyways now. This is the current congif:
```
"prettier": "prettier './**/*.js' './**/*.sol'"
```

Reply:
After looking further into this this it looks like this only happens with my prettier in my vs code and not if I run `npm run lint` or `npm run prettier`. Both of these commands respect the config in the `package.json`. However linting from my editor results in _all_ markdown files being linted if they are open in my browser.

I'd say we have 2 options:
1) add `*.md` to the `.prettierignore` to resolve this consistently for everyone to disable linting for documentation.
2) lint our markdown! Prettier can do this in our current setup so it's not too big a leap to get everyone to lint the docs as well. 

I like idea 2 🔥

Reply:
I'm fine with linting markdown

Reply:
SGTM

Reply:
Let's remove this stuff from this PR, though, and add the linting in a separate PR.

Reply:
done, took me a while to figure out that I needed to `--no-verify` my commit in order to get changes back. FYI the command I used was the nifty `git checkout origin/master -- path/to/file/`